packages/generator/tests/acceptance/test_non_streaming_measurement_client.py

```diff
@@ -33,19 +36,45 @@ def test___measurement_plugin_client___measure___returns_output(
     assert measure_output == expected_measure_output
 
 
+def test___measurement_plugin_client___stream_measure___returns_output(
+    measurement_plugin_client_module: ModuleType,
+) -> None:
+    test_measurement_client_type = getattr(measurement_plugin_client_module, "TestMeasurement")
+    output_type = getattr(measurement_plugin_client_module, "Output")
+    expected_measure_output = output_type(
+        float_out=0.05999999865889549,
+        double_array_out=[0.1, 0.2, 0.3],
+        bool_out=False,
+        string_out="sample string",
+        string_array_out=["String1", "String2"],
+        path_out="path/test",
+        path_array_out=["path/test1", "path/ntest2"],
+        io_out="resource",
+        io_array_out=["resource1", "resource2"],
+        integer_out=10,
+        xy_data_out=None,
+    )
+    measurement_plugin_client = test_measurement_client_type()
+
+    measure_output = measurement_plugin_client.stream_measure()
+
+    for output in measure_output:
+        assert output == expected_measure_output
```

Comment:
The non-streaming measurement should return exactly one response. This will pass if the stream returns too few or too many responses.

Do something like this:
```
    responses = [response for response in response_iterator]
    assert len(responses) == 1
    assert responses[0] == expected_measure_output
```

Reply:
Sure, updated as suggested.